inc/commonitilobject.class.php

```diff
@@ -7162,6 +7162,20 @@ function showTimeline($rand) {
                   ]);
                }
             }
+
+            if ($objType == 'Ticket' && $item['type'] == TicketTask::getType()) {
+               if (isset($item_i['sourceof_items_id']) && $item_i['sourceof_items_id'] > 0) {
+                  echo Html::link('', Ticket::getFormURLWithID($item_i['sourceof_items_id']), [
+                     'class' => 'fa fa-code-branch control_item disabled',
+                     'title' => __('Task was already promoted')
+                  ]);
+               } else {
+                  echo Html::link('', Ticket::getFormURL() . "?_promoted_task_id=" . $item_i['id'], [
+                     'class' => 'fa fa-code-branch control_item',
+                     'title' => __('Promote to Ticket')
+                  ]);
+               }
+            }
```

Comment:
As we are currently redoing the ui in feat/modern-ui branch, all this part is completly incompatible (and will cause merging issues)
I agree with the proposal but we will redo it (before the end of the week) to be done on the mentioned branch.
So consider our acceptation for your work, but the current pr will be closed just after